source/ingress.go

```diff
@@ -112,11 +112,15 @@ func getEndpointsFromTargetAnnotation(ing *v1beta1.Ingress, hostname string) []*
 	// Get the desired hostname of the ingress from the annotation.
 	targetAnnotation, exists := ing.Annotations[targetAnnotationKey]
 	if exists {
+		ttl, err := getTTLFromAnnotations(ing.Annotations)
+		if err != nil {
+			log.Warn(err)
+		}
```

Comment:
Since you continue the loop here we should set `ttl` to something sane, e.g. `300`.

Reply:
i think it is okay to leave it as it is here, as this should be handled in providers (if they care about TTL field in endpoint)